content/zh/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -150,6 +156,91 @@ Envoy 中的全局速率限制使用 gRPC API 向速率限制服务请求配额
     EOF
     {{< /text >}}
 
+### 全局速率限制高级示例  {#global-rate-limit-advanced-case}
+
+此示例使用正则表达式来匹配 `/api/*` `uri`
+并使用 VirtualService http 名称定义在路由级别插入的速率限制操作。
+前面示例中插入的 PATH 值 `api` 开始发挥作用。
+
+1. 更改 VirtualService，前缀 `/api/v1/products` 被移动到名为 `api` 的路由中：
+
+{{< text bash >}}
```

Comment:
The original code block indents incorrectly.

Reply:
Fixed.